src/main/java/core/basesyntax/service/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String operation;
+
+    Operation(String operation) {
+        this.operation = operation;
+    }
+
+    public String getOperation() {
+        return operation;
+    }
+
+    public static Operation fromString(String operation) {
+        for (Operation operations : Operation.values()) {
+            if (operations.operation.equalsIgnoreCase(operation)) {
+                return operations;
+            }
+        }
+        throw new IllegalArgumentException("No constant with operation " + operation + " found");
```

Comment:
same here, you have to come up with a scenario where this exception is thrown
or just skip this class altogether in pom.xml